Admin/Admin.php

```diff
@@ -897,6 +904,11 @@ public function buildDatagrid()
         foreach ($this->getExtensions() as $extension) {
             $extension->configureDatagridFilters($mapper);
         }
+
+        // set template for SimplePager if it's not already overloaded
+        if ($this->getPagerType() === 'simple' && $this->templates['pager_results'] === 'SonataAdminBundle:Pager:results.html.twig') {
```

Comment:
Indeed, this is just a small help for user to not need to change the pager template if he change the pager type. Any better ideas are welcome. Inside dependency injection is too sooner for this (the pager type is unknown).